### DISCUSSION TITLE
v2ray 5.x for windows移除了wv2ray.exe，那么请问有什么参数或者配置，可以让主程序在后台运行吗？

### DISCUSSION THREAD

**Discussion by vcmars** [v2fly/v2ray-core#2397] [Q&A]
我尝试了：
**start /b v2ray.exe** 
以及Sysinternals Suite 中的 PsExec 工具：
**psexec -d v2ray.exe**

都无效，v2ray.exe都会顽固的跳出来显示console信息。

**Comment by nursery01** [comment]
新建文檔  run.vbe
```
set ws=wscript.createobject("wscript.shell")
ws.run "v2ray.exe /start",0
```
雙擊 run.vbe

**Comment by mydogshitgold** [comment]
wv2ray.exe use the GUI subsystem, and can be easily built with `-H windowsgui` flag.

```
git clone --depth 1 https://github.com/v2fly/v2ray-core
cd v2ray-core
set CGO_ENABLED=0
go build -o wv2ray.exe -trimpath -ldflags "-s -w -H windowsgui -buildid=" ./main
```

Depending on your network speed, the build process should take less than 2 minutes.

**Comment by vcmars** [comment]
我已自行找到办法，使用powershell加隐藏窗口的方式拉起运行V2ray 5.x：

`powershell.exe -Command "Start-Process 'v2ray5.exe' 'run -d conf' -WindowStyle Hidden"`
